lib/bindings/python/rust/llm/kv.rs

```diff
@@ -191,6 +198,71 @@ impl ZmqKvEventPublisher {
     }
 }
 
+/// A ZMQ-based key-value cache event listener that operates independently
+/// of the dynamo runtime or event plane infrastructure.
+#[pyclass]
+pub(crate) struct ZmqKvEventListener {
+    event_receiver: Arc<tokio::sync::Mutex<tokio::sync::mpsc::UnboundedReceiver<KvCacheEvent>>>,
+    shutdown_token: tokio_util::sync::CancellationToken,
+}
+
+#[pymethods]
+impl ZmqKvEventListener {
+    #[new]
+    fn new(zmq_endpoint: String, zmq_topic: String, kv_block_size: usize) -> PyResult<Self> {
```

Comment:
is `kv_block_size == 0` supported? I doubt it would fail, but at least logging an unexpected value could be helpful for anyone interacting w/ this code.

if this point isn't reachable by a third-party utilizing this functionality, then I guess I don't care -- but if this basically the backing of an API, then we have to care.